.github/workflows/hosted.yml

```diff
@@ -63,16 +63,36 @@ jobs:
       with:
         maven-version: 3.9.4
 
-    - name: Build ANTLR with Maven
+    - name: Build ANTLR modules without testing
       run: mvn install -DskipTests=true -Darguments="-Dmaven.javadoc.skip=true" -B -V
 
+
+    - name: Unit test core module
+      if: matrix.target != 'tool'
+      run: |
+        cd runtime/Core
+        mvn test
+
+    - id: capitalize
+      name: Capitalize target module name
```

Comment:
Why capitalization is required?

Reply:
Because the target name is lowercase but the subdirectory name is capitalized I.e. kotlin vs Kotlin . In the end state we probably want to normalize that but that's a big change requiring a dedicated PR